project_task_description_template/views/task_templates.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
**suggestion:** Split this into two separate files: `project_views.xml` and `project_task_template_views.xml`, and move the inherited views into the first file and the new views into the second.

Reply:
Thanks, I'll fix it